src/pages/Search/Search.js

```diff
@@ -279,18 +395,21 @@ export default class Search extends Component {
                     onChange={this.handleChange2}
                     options={this.state.programs}
                   />
-                   <div className="headingRow"> Number of Sessions </div>
+                  <div className="headingRow"> Number of Sessions </div>
                   <Select className="dropdown"
                     value={sessions}
                     onChange={this.handleChange5}
                     options={options5}
                   />
-                  <div className="headingRow"> Day of the Week </div> 
+                  <div>
+                  <div className="headingRow"> Select Method to select dates </div>
                   <Select className="dropdown"
-                    value={daysOfWeek}
-                    onChange={this.handleChange8}
-                    options={daysOfWeekOptions}
+                    value={selectDates}
+                    onChange={this.handleDateOptionSelection}
+                    options={selectDatesOptions}
                   />
+                  </div>
+                  {this.handleDateOptionDisplay(selectDates.value)}
```

Comment:
The default value of `selectDates.value` is `dayOfWeek` which causes a popup. So when a user first navigates to this page, the popup is shown.
<img width="1624" alt="Screenshot 2019-07-15 14 32 08" src="https://user-images.githubusercontent.com/10586811/61239630-5e4c8f80-a70d-11e9-8c78-f1910f1ec9a6.png">